PetAdoptions/pethistory/template.yaml

```diff
@@ -0,0 +1,215 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Transform: AWS::Serverless-2016-10-31
+Description: Pet History Lambda function for processing adoption history messages from SQS
+
+Parameters:
+  SQSQueueArn:
+    Type: String
+    Default: "arn:aws:sqs:REGION:ACCOUNTID:QUEUE_NAME"
+    Description: ARN of the SQS queue for adoption messages
+
+  RDSSecretArn:
+    Type: String
+    Default: "arn:aws:secretsmanager:REGION:ACCOUNTID:secret:SECRET_NAME"
+    Description: ARN of the RDS secret in Secrets Manager
+
+  VpcId:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Parameter VpcId not used.